client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningInfo/useReachContact.ts

```diff
@@ -12,7 +12,7 @@ const useReachContact = (props: Props) => {
     const { alertWarning , alertError } = useCustomSwal();
 
     const formHasErrors = errors.form ? Boolean(errors.form[index]) : false;
-
+    const isInterrogationOfContactsHaveCity = formValues.isolationAddress?.city !== null && formValues?.isolationAddress.city !==  undefined
```

Comment:
I think you can use Boolean(formValues?.isolationAddress?.city)

Reply:
same for line 57 in useContactQuestioning.ts